src/Sparkfabrik/Command/Redmine/RedmineUpdateCommand.php

```diff
@@ -0,0 +1,226 @@
+            'issue',
+            InputArgument::REQUIRED,
+            'Issue id'
+        );
+        $this->addOption(
+            'project',
+            'p',
+            InputOption::VALUE_OPTIONAL,
+            'The project the issue must belong to.'
+        );
+        $this->addOption(
+            'tracker',
+            null,
+            InputOption::VALUE_OPTIONAL,
+            'The new tracker to set for the issue.'
+        );
+        $this->addOption(
+            'status',
+            null,
+            InputOption::VALUE_OPTIONAL,
+            'The new status for the issue.'
+        );
+        $this->addOption(
+            'priority',
+            null,
+            InputOption::VALUE_OPTIONAL,
+            'The new priority for the issue.'
+        );
+        $this->addOption(
+            'category',
+            null,
+            InputOption::VALUE_OPTIONAL,
+            'The new category for the issue.'
+        );
+        $this->addOption(
+            'subject',
+            null,
+            InputOption::VALUE_OPTIONAL,
+            'The new subject to set for the issue.'
+        );
+        $this->addOption(
+            'description',
+            null,
+            InputOption::VALUE_OPTIONAL,
+            'The new description to set for the issue. *Use with care, odds are good what you really want is to add a comment!*'
+        );
+        $this->addOption(
+            'target-version',
+            null,
+            InputOption::VALUE_OPTIONAL,
+            'The new target version for the issue (id).'
+        );
+        $this->addOption(
+            'assignee',
+            null,
+            InputOption::VALUE_OPTIONAL,
+            'The new person in charge for the issue (id).'
+        );
+        $this->addOption(
+            'notes',
+            null,
+            InputOption::VALUE_OPTIONAL,
+            'A note to add to the issue (comments or description for the update).'
+        );
+        $this->addOption(
+            'custom-field',
+            null,
+            InputOption::VALUE_OPTIONAL | InputOption::VALUE_IS_ARRAY,
+            'A custom field in the "name:value" or "id:value" format.'
+        );
+        $this->addOption(
+            'dump-issue',
+            null,
+            InputOption::VALUE_NONE,
+            'Dump the issue information on successful update as for `redmine:show` command with `--complete` option.'
+        );
+    }
+
+    /**
+     * Returns a properly formatted custom fields array.
+     */
+    protected function parseCustomFields($customFieldPairs) {
+        $customFields = [];
+        foreach ($customFieldPairs as $customFieldPair) {
+            $customFields[] = $this->formatCustomFieldFromPair($customFieldPair);
+        }
+
+        // Clean up all missing keys
+        array_filter($customFields, function($v) {
+           return $v['name'] !== '>>>SPARK-DELETE<<<';
+        });
+
+        // Clean up malformed pairs such as ":3" or ":"
```

Comment:
Mind that `4:` or `Browser:` pairs are legit, since you could want to clear a field content.